cartography/intel/aws/resourcegroupstaggingapi.py

```diff
@@ -41,6 +41,11 @@ def get_bucket_name_from_arn(bucket_arn):
     'ec2:security-group': {'label': 'EC2SecurityGroup', 'property': 'id', 'id_func': get_short_id_from_ec2_arn},
     'ec2:subnet': {'label': 'EC2Subnet', 'property': 'subnetid', 'id_func': get_short_id_from_ec2_arn},
     'ec2:vpc': {'label': 'AWSVpc', 'property': 'id', 'id_func': get_short_id_from_ec2_arn},
+    'ec2:transit-gateway': {'label': 'AWSTransitGateway', 'property': 'id', 'id_func': get_short_id_from_ec2_arn},
+    'ec2:transit-gateway-attachment': {
+        'label': 'AWSTransitGatewayAttachment', 'property': 'id',
+        'id_func': get_short_id_from_ec2_arn,
```

Comment:
Similar idea here: we should remove `id_func`.

In fact, I don't think `get_short_id_from_ec2_arn()` was meant for strings like the TGW IDs here. There's no observed bug because `get_short_id_from_ec2_arn("tgw-0123456789abcdef0")` will still return `"tgw-0123456789abcdef0"`, but if we're not careful in the future another PR could introduce a side effect and produce a hard to track down bug.

Reply:
Will do